### PR TITLE
Adds description for capacity

### PR DESCRIPTION
**Pull request by develaper**

### CODE REVIEW

**Review by alexeh — APPROVED** [review]

**Review by agnessa — COMMENTED** [review]
Regarding decision making, perhaps this role should be always involved in the advice process for hiring & outsourcing.

File: _docs/backenders/strategy-2021/roles/capacity.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+**Current responsible**: -
+
+## Responsibility:
+    **Meetings and other FA:**
+- Facilitator on the capacity part of the BE SMO meeting.
+- Attend to capacity monthly meeting.
+- Participate in fortnightly BD meetings.
+- Participate in BD/FE retrospectives every 6 months.
+- Point of contact for other functional areas.
```
Comment (develaper) [inline _docs/backenders/strategy-2021/roles/capacity.md:18]: @tiagojsag I know you added a note in the excel about this point. Would you like to have a more clear description for this responsability?

File: _docs/backenders/strategy-2021/roles/capacity.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    **Inside the team:**
+- Having a track on the project proposal pipe.
+- Map and communicate skills.
+- Improve communication of medium and long-term capacity estimations.
+- Listen and look for solutions to the group needs on capacity wise making sure capacity needs from the BE group are covered.
+- Push people to participate on capacity wise and ensure that, as a team we are aware and keep track of current allocations and delayed projects.
+- Improve the proposals allocation workflow and facilitate allocating people required if needed based on availability, interests and skills. 
+
+Not clear:
+Drive and promote automation for data collection and modelling predictions.
```
Comment (develaper) [inline _docs/backenders/strategy-2021/roles/capacity.md:29]: @aagm Shall we refactor this sentence to make it sound less complicated and/or ambitious?

File: _docs/backenders/strategy-2021/roles/capacity.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+***
+
+
+## Impact:
+- Detect possible lacks on capacity for the future.
+- Detect in advance possible spare capacity.
+- All projects will be covered on capacity wise.
+	- Too ambitious given that we cannot control all the moving parts. So here we need a more clear definition/approach to what does it mean and how much can we cover.
```
Comment (develaper) [inline _docs/backenders/strategy-2021/roles/capacity.md:37]: @hotzevzl any suggestions on how to rephrase it to fit our range of control?

File: _docs/backenders/strategy-2021/roles/capacity.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+- Attend to capacity monthly meeting.
+- Participate in fortnightly BD meetings.
+- Participate in BD/FE retrospectives every 6 months.
+- Point of contact for other functional areas.
+
+    **Inside the team:**
+- Having a track on the project proposal pipe.
+- Map and communicate skills.
+- Improve communication of medium and long-term capacity estimations.
+- Ensure alighnment between the capacity needs, hiring and outsourcing processes.
```
Comment (develaper) [inline _docs/backenders/strategy-2021/roles/capacity.md:24]: SOmething like that @agnessa ?

File: _docs/backenders/strategy-2021/roles/capacity.md (hunk 1 above)
Comment (tiagojsag) [reply _docs/backenders/strategy-2021/roles/capacity.md:18]: Point of contact for other functional areas to request occasional support on small tasks (i.e.: DNS changes, restart servers, legacy project related work, etc).

File: _docs/backenders/strategy-2021/roles/capacity.md (hunk 4 above)
Comment (agnessa) [reply _docs/backenders/strategy-2021/roles/capacity.md:24]: I leave it for @aagm to decide, given that she has bravely volunteered :)

File: _docs/backenders/strategy-2021/roles/capacity.md (hunk 2 above)
Comment (aagm) [reply _docs/backenders/strategy-2021/roles/capacity.md]: i had pushed some changes there.

**Review by aagm — APPROVED** [review]

File: _docs/backenders/strategy-2021/roles/capacity.md [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+parent: Roles
+grand_parent: Strategy 2021
+---
+
+# Capacity role definition
+
+**Current responsible**: Alicia A. Arenzana
+
+## Responsibility:
+    **Meetings and other FA:**
```
Comment (ikas) [inline _docs/backenders/strategy-2021/roles/capacity.md:13]: I believe you need to remove the tab at the start of the line, otherwise this is printed like a code snippet :)

File: _docs/backenders/strategy-2021/roles/capacity.md [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+- Facilitator on the capacity part of the BE SMO meeting.
+- Attend to capacity monthly meeting.
+- Participate in BD/FE retrospectives every 6 months.
+- Don't leave unanswered other functional areas requests for occasional support on small tasks (i.e.: DNS changes, restart servers, legacy project related work, etc).
```
Comment (ikas) [inline _docs/backenders/strategy-2021/roles/capacity.md:17]: I'd frame this in a more positive way, like: Answer other functional area requests for occasional support...

File: _docs/backenders/strategy-2021/roles/capacity.md [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+
+    **Inside the team:**
+- Track with BD representative the project proposal pipe so we can anticipate and forecast capacity needs. in order to do so:
```
Comment (ikas) [inline _docs/backenders/strategy-2021/roles/capacity.md:20]:
```suggestion
- Track with BD representative the project proposal pipeline so we can anticipate and forecast capacity needs. In order to do so:
```

File: _docs/backenders/strategy-2021/roles/capacity.md [hunk 8]
```diff
@@ -0,0 +1,43 @@
...
+    - Work in conjunction with the other FA capacity representatives in order to model and standardize of FA capacity status and predictions.
```
Comment (ikas) [inline _docs/backenders/strategy-2021/roles/capacity.md:21]:
```suggestion
    - Work in conjunction with the other FA capacity representatives to model and standardize the FA capacity status and predictions.
```

File: _docs/backenders/strategy-2021/roles/capacity.md [hunk 9]
```diff
@@ -0,0 +1,43 @@
...
+    - Work with Miguel Mendoza to extract relevant information from vizz-tracker.
+    - Work in conjunction with other FA in order to automate and standardize data collection.
+- Map (adhere to the same tool used by frontend), track and communicate skills if the knowledge and mentorship rol has no representative.
```
Comment (ikas) [inline _docs/backenders/strategy-2021/roles/capacity.md:24]:
```suggestion
- Map (adhere to the same tool used by the frontend), track and communicate skills if the knowledge and mentorship role has no representative.
```

File: _docs/backenders/strategy-2021/roles/capacity.md [hunk 10]
```diff
@@ -0,0 +1,43 @@
...
+- Improve communication of medium and long-term Backend capacity estimations.
+- Track and ensure alignment between the capacity needs, hiring and outsourcing processes.
+- Listen and work together with the BE team to find solutions to the needs on capacity and make sure capacity needs from the BE group are covered.
+- Push people to participate on capacity wise and ensure that, as a team we are aware and keep track of current allocations and delayed projects, as well as capacity imbalance are reduce to the bare minimum.
+- Improve the proposals allocation workflow and facilitate allocating people required if needed based on availability, interests and skills. 
+***
```
Comment (ikas) [inline _docs/backenders/strategy-2021/roles/capacity.md:30]: These `***` are not needed

File: _docs/backenders/strategy-2021/roles/capacity.md [hunk 11]
```diff
@@ -0,0 +1,43 @@
...
+
+
+## Impact:
+- Detect possible lacks on capacity for the future.
+- Detect in advance possible spare capacity.
+- All projects will be covered on capacity wise, based on the available information (BD meetings and capacity spreadsheet && meetings, team members on delayed projects).
+- BE Team members feel supported and their demands, capacity wise, are listened and an action is taken.
+***
```
Comment (ikas) [inline _docs/backenders/strategy-2021/roles/capacity.md:38]: same here

**Review by ikas — COMMENTED** [review]
Minor tweaks, otherwise LGTM :+1:

**Review by tiagojsag — APPROVED** [review]

**Review by hotzevzl — APPROVED** [review]

**Review by alexeh — APPROVED** [review]

**Review by aagm — APPROVED** [review]
